### ISSUE TITLE
az aks create shows outdated help text for LB / VM settings

### ISSUE THREAD

**Issue by jason-epp** [Azure/AKS#1371]
Previously, we were creating clusters without specifying --load-balancer-sku, and a basic LB was created.  This was the case as of Oct 8 '19.  
Now it is creating a Standard LB

I don't see a default value defined for --load-balancer-sku in the documentation...

https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-create

and it kind of implies that 'Basic' is the default...

```
Create a kubernetes cluster with default kubernetes version, default SKU load balancer(basic) and default vm set type(AvailabilitySet).
az aks create -g MyResourceGroup -n MyManagedCluster
```

**Comment by plytro** [comment]
https://github.com/Azure/AKS/releases/tag/2019-11-11

This change was detailed in the release notes in November.

**Comment by plytro** [comment]
https://github.com/Azure/azure-cli/issues/11642

**Comment by jason-epp** [comment]
what if we havn't update to that version of az?

~$ az --version
azure-cli                         2.0.75 *

command-modules-nspkg              2.0.3
core                              2.0.75 *
nspkg                              3.0.4
telemetry                          1.0.4

Extensions:
aks-preview                        0.4.5
azure-devops                       0.7.0
log-analytics                      0.1.3

Python location '/opt/az/bin/python3'
Extensions directory '/var/lib/jenkins/.azure/cliextensions'

Python (Linux) 3.6.5 (default, Oct 11 2019, 09:04:16)
[GCC 5.4.0 20160609]

Legal docs and information: aka.ms/AzureCliLegal

**Comment by jluk** [comment]
@jason-epp for that old CLI build it should still deploy basic lb / vmas by default. is it deploying the new configs when you try? it's also possible the preview extension impacts the settings, so it's worth trying with preview turned off.

Regardless the best path forward is to update to latest and specify the exact config you wish to use if you want to use older defaults.

I'm updating the text and adding tags to add visibility to the help text bug https://github.com/Azure/azure-cli/issues/11642 as mentioned by @plytro as we do need to update that. Thanks for the report.

**Comment by jason-epp** [comment]
Well, that version is deploying a std LB.
We had to change our scripts to include --load-balancer-sku set to 'basic' to force it to deploy a basic LB.

I want to point out that the default operations should not change when adding new functionality and options.   Especially considering it changes to use the paid option.  Many examples and scripts out in the wild are now rendered invalid and will deploy the option that includes a cost.  Leading many who are trying AKS out to incur a needless cost.

Also, the next step in building a cluster with an nginx ingress is to create and assign an IP.  And the default option for that is a basic IP, and that functionality hasn't changed.  That's how we found this.  Scripts that previously worked were now failing because of the sku mismatch (std LB with basic IP).

**Comment by jason-epp** [comment]
I am sorry, I made a huge mistake.

I was on the wrong vm when I ran az --version.  Our deployments scripts were moved to a new machine, and I was on the old one.  

The new one has...

azure-cli                         2.0.78 *

command-modules-nspkg              2.0.3
core                              2.0.78 *
nspkg                              3.0.4
telemetry                          1.0.4

Python location '/opt/az/bin/python3'
Extensions directory '/home/jepp/.azure/cliextensions'

Python (Linux) 3.6.5 (default, Dec 12 2019, 11:11:32)
[GCC 5.4.0 20160609]

Legal docs and information: aka.ms/AzureCliLegal

Sorry for the confusion.
